static/css/devhub/new-landing/sections/more-information.less

```diff
@@ -29,6 +29,7 @@
   ul {
     list-style: none;
     padding: 0;
+    width: 90%;
```

Comment:
Hmm, good point. Now that I'm looking at it a bit more closely I notice that the `div` around those `ul`s does have a proper padding set. Let me dig into this a bit deeper early tomorrow morning with a fresh mind